target/linux/realtek/files-5.10/arch/mips/rtl838x/prom.c

```diff
@@ -29,10 +29,34 @@ extern const char __appended_dtb;
 struct rtl83xx_soc_info soc_info;
 const void *fdt;
 
+#ifdef CONFIG_MIPS_MT_SMP
+extern const struct plat_smp_ops vsmp_smp_ops;
+
+static struct plat_smp_ops rtl_smp_ops;
```

Comment:
You can reduce the scope of this variable and move it before the line:
`rtl_smp_ops = vsmp_smp_ops;`

Reply:
Done